cmd/podman/commit.go

```diff
@@ -120,7 +121,17 @@ func commitCmd(c *cli.Context) error {
 		Changes:       c.StringSlice("change"),
 		Author:        c.String("author"),
 	}
-	newImage, err := ctr.Commit(getContext(), reference, options)
+	var createArtifact createConfig
+	artifact, err := ctr.GetArtifact("create-config")
+	if err == nil {
+		if err := json.Unmarshal(artifact, &createArtifact); err != nil {
+			return nil
```

Comment:
any value to adding a log.debug to display more info on error?

Reply:
actually this is a hard failure ... adding return err